doc/whats_new/v0.20.rst

```diff
@@ -91,6 +91,8 @@ Model evaluation
 - Added the :func:`metrics.balanced_accuracy_score` metric and a corresponding
   ``'balanced_accuracy'`` scorer for binary classification.
   :issue:`8066` by :user:`xyguo` and :user:`Aman Dalmia <dalmia>`.
+- Added the :func:`metrics.mean_absolute_percentage_error` metric and the associated
```

Comment:
"Model evaluation" is not the best section for this I would say, in doc/whats_new/v0.19.0.rst there is a "Metrics" section. I think you can do the same here.

Reply:
Nice catch, I will do that.